src/ui/TreeView.js

```diff
@@ -50,8 +50,8 @@ class TreeView {
     return yo`<ul style=${this.cssList}>${children}</ul>`
   }
 
-  formatDataDefault (key, self, children, expand) {
-    var label = yo`<span style=${this.cssLabel}><label style='width: 10px'></label><label>${key}: ${self}</label></span>`
+  formatDataInternal (key, data, children, expand) {
```

Comment:
Hm, I would swap the naming here, `formatData` is kind of more internal than `formatDataInternal` ;-)
What about: `formatData` for the general thing and `formatSelf` for what is called `formatData` in your PR?